Clarifying which command to use to convert bots (#4953)
mattermost/docs: source/developer/bot-accounts.rst

How do I convert an existing account to a bot account?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> To use `mmctl`<https://docs.mattermost.com/manage/mmctl-command-line-tool.html> to convert a user to a bot use this command:

Reviewer: ```suggestion
You can use the `mmctl command line tool `<https://docs.mattermost.com/manage/mmctl-command-line-tool.html>`_ to convert a user to a bot, with the following command:
```

.. code-block:: text

  mmctl user convert user@example.com --bot

> In addition to email, you may identify the user by its username or user ID.

Reviewer: ```suggestion
In addition to an email address, users can be identified by their Mattermost username or user ID.
```
